src/mcp/tools/project/list_apps.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint camelcase: 0 */
+
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { AppPlatform, listFirebaseApps } from "../../../management/apps.js";
+
+const PLATFORM_MAP: Record<string, AppPlatform> = {
```

Comment:
Repeated in a few places - wanna just put this in src/management/apps.js?

Reply:
Decided to just rework this a little to avoid needing the map.